src/pages/Home/Home.jsx

```diff
@@ -1,10 +1,198 @@
+import { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import { Button } from '@/components/Button';
+import MenuList from '@/components/Menu/MenuList';
+import supabase from '@/supabase/supabaseClient';
+
 const Home = () => {
+  const [menus, setMenus] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    const loadMenus = async () => {
+      try {
+        const { data, error } = await supabase.from('menus').select('title, content, thumbnail, category').limit(4);
+
+        if (error) throw new Error(error.message);
+
+        setMenus(data);
+      } catch (error) {
+        console.error('Error loading menus:', error);
+        setIsError(true);
+      } finally {
+        setIsLoading(false);
+      }
+    };
+
+    loadMenus();
+  }, []);
+
+  if (isLoading) return <Loading>Loading...</Loading>;
+  if (isError) return <Error>Error loading menus</Error>;
+
   return (
-    <img
-      src="https://images.unsplash.com/photo-1718524767487-7f98d4e5eeeb?q=80&w=3174&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D"
-      alt=""
-    />
+    <StContainer>
+      <StMainSection>
+        <StIntroSection>
+          <StIntroText1>
+            1943: 전통과 현대가
+            <br />
+            어우러진 공간
+          </StIntroText1>
+          <StIntroText2>
+            1943 Drink, Laugh, and Immerse Yourself
+            <br />
+            in a Classic Atmosphere
+          </StIntroText2>
+          <Button as={Link} to="/store" aria-label="Find Store">
```

Comment:
```suggestion
          <Button href="/store" aria-label="Find Store">
```
이렇게 하면 자동으로 버튼 컴포넌트가 Link 컴포넌트를 사용합니다!